index.html

```diff
@@ -53,6 +53,120 @@ <h1>I’m Seko<br> Glad to <br>see you!</h1>
+                    </ul>
+                    <button class="btn btn-primary">See Project</button>
+                </div>
+            </div>
+
+            <div class="work-container">
+                <div class="work-image">
+                    <img src="/img/gym fit.png" width="278px" height="220px" alt="">
+                </div>
+                <div class="work-content">
+                    <h2>SPiiKA French</h2>
+                    <ul class="work">
+                        <li class="work-item works-title">Language</li>
+                        <li class="work-item works-role"> <img src="/img/Counter.png" alt=""> Back End Dev</li>
+                        <li class="work-item works-year"> <img src="/img/Counter.png" alt=""> 2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="techs">
+                        <li class="btn btn-secondary">html</li>
+                        <li class="btn btn-secondary">css</li>
+                        <li class="btn btn-secondary">javascript</li>
+                    </ul>
+                    <button class="btn btn-primary">See Project</button>
+                </div>
+            </div>
+
+            <div class="work-container">
+                <div class="work-image">
+                    <img src="/img/art printing.png" width="278px" height="220px" alt="">
+                </div>
+                <div class="work-content">
+                    <h2>Professional Art</h2>
+                    <ul class="work">
+                        <li class="work-item works-title">Art</li>
+                        <li class="work-item works-role"> <img src="/img/Counter.png" alt=""> Back End Dev</li>
+                        <li class="work-item works-year"> <img src="/img/Counter.png" alt=""> 2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="techs">
+                        <li class="btn btn-secondary">html</li>
+                        <li class="btn btn-secondary">css</li>
+                        <li class="btn btn-secondary">javascript</li>
+                    </ul>
+                    <button class="btn btn-primary">See Project</button>
+                </div>
+            </div>
+
+
+        <section class="about-section">
+            <div class="about-container">
+                <div class="desc">
+                    <h2 class="about-heading">About<br> Myself</h2>
+                    <p class="about-desc">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href="#" class="connect">Let's Connect</a>
```

Comment:
- [ ] Please note that the `Let's Connect` title should be a link, instead it should be a `hx` title for the icons below it , kindly fix it

Reply:
please I do not understand this pretty well. ==>"Please note that the Let's Connect title should be a link, instead it should be a hx title for the icons below it, kindly fix it"